<?xml version="1.0" encoding="UTF-8"?><testrun name="test" project="Assignment 5" tests="59" started="59" failures="0" errors="10" ignored="0">
  <testsuite name="com.usu.sudokuSolver.utils.test.UtilitiesTest" time="3.095">
    <testcase name="testExportSolution_NullBoard" classname="com.usu.sudokuSolver.utils.test.UtilitiesTest" time="0.002"/>
    <testcase name="testExportSolution_AllCorrect" classname="com.usu.sudokuSolver.utils.test.UtilitiesTest" time="0.007"/>
    <testcase name="testExportSolution_EmptyBoard" classname="com.usu.sudokuSolver.utils.test.UtilitiesTest" time="0.008"/>
    <testcase name="testExportSolution_EmptyFileName" classname="com.usu.sudokuSolver.utils.test.UtilitiesTest" time="3.067"/>
    <testcase name="testExportSolution_NullFileName" classname="com.usu.sudokuSolver.utils.test.UtilitiesTest" time="0.011"/>
  </testsuite>
  <testsuite name="com.usu.sudokuSolver.board.test.SudokuBoardTest" time="0.062">
    <testcase name="testRows" classname="com.usu.sudokuSolver.board.test.SudokuBoardTest" time="0.009"/>
    <testcase name="testGetCell_Correct" classname="com.usu.sudokuSolver.board.test.SudokuBoardTest" time="0.007"/>
    <testcase name="testSetCell_NullValue" classname="com.usu.sudokuSolver.board.test.SudokuBoardTest" time="0.006"/>
    <testcase name="testColumns" classname="com.usu.sudokuSolver.board.test.SudokuBoardTest" time="0.011"/>
    <testcase name="testSetCell_SymbolNullValue" classname="com.usu.sudokuSolver.board.test.SudokuBoardTest" time="0.009"/>
    <testcase name="testGetCell_NullValue" classname="com.usu.sudokuSolver.board.test.SudokuBoardTest" time="0.005"/>
    <testcase name="testSetCell" classname="com.usu.sudokuSolver.board.test.SudokuBoardTest" time="0.015"/>
  </testsuite>
  <testsuite name="com.usu.sudokuSolver.board.test.SudokuSolver4x4Test" time="0.085">
    <testcase name="testSizeColumn_CorrectValue" classname="com.usu.sudokuSolver.board.test.SudokuSolver4x4Test" time="0.021"/>
    <testcase name="testIsSolved_NullValues" classname="com.usu.sudokuSolver.board.test.SudokuSolver4x4Test" time="0.016"/>
    <testcase name="testIsSolved_Correct" classname="com.usu.sudokuSolver.board.test.SudokuSolver4x4Test" time="0.013"/>
    <testcase name="testSizeRow_CorrectValue" classname="com.usu.sudokuSolver.board.test.SudokuSolver4x4Test" time="0.001"/>
    <testcase name="testIsSolved_NullValues_WrongValue" classname="com.usu.sudokuSolver.board.test.SudokuSolver4x4Test" time="0.0"/>
    <testcase name="testContainOutlier_CorrectValue" classname="com.usu.sudokuSolver.board.test.SudokuSolver4x4Test" time="0.0"/>
    <testcase name="testContainOutlier_NullValue" classname="com.usu.sudokuSolver.board.test.SudokuSolver4x4Test" time="0.0"/>
    <testcase name="testContainDuplicate_NullValue" classname="com.usu.sudokuSolver.board.test.SudokuSolver4x4Test" time="0.005"/>
    <testcase name="testIsSolved_Correct_WrongValue" classname="com.usu.sudokuSolver.board.test.SudokuSolver4x4Test" time="0.001"/>
    <testcase name="testContainDuplicate_CorrectValue" classname="com.usu.sudokuSolver.board.test.SudokuSolver4x4Test" time="0.002"/>
    <testcase name="testContainDuplicateRow_CorrectValue" classname="com.usu.sudokuSolver.board.test.SudokuSolver4x4Test" time="0.0"/>
    <testcase name="testContainDuplicateRow_NullValue" classname="com.usu.sudokuSolver.board.test.SudokuSolver4x4Test" time="0.0"/>
  </testsuite>
  <testsuite name="com.usu.sudokuSolver.board.test.SudokuBoardParserTest" time="0.113">
    <testcase name="testParsingLogic_NullInput" classname="com.usu.sudokuSolver.board.test.SudokuBoardParserTest" time="0.0"/>
    <testcase name="testParsingLogic_Correct" classname="com.usu.sudokuSolver.board.test.SudokuBoardParserTest" time="0.088"/>
    <testcase name="testParsingLogic_EmptyInput" classname="com.usu.sudokuSolver.board.test.SudokuBoardParserTest" time="0.025"/>
  </testsuite>
  <testsuite name="com.usu.sudokuSolver.board.test.SudokuSolver16x16Test" time="0.221">
    <testcase name="testSizeColumn_CorrectValue" classname="com.usu.sudokuSolver.board.test.SudokuSolver16x16Test" time="0.015"/>
    <testcase name="testIsSolved_NullValues" classname="com.usu.sudokuSolver.board.test.SudokuSolver16x16Test" time="0.012"/>
    <testcase name="testContainDuplicateColumn_CorrectValue" classname="com.usu.sudokuSolver.board.test.SudokuSolver16x16Test" time="0.011"/>
    <testcase name="testSizeRow_CorrectValue" classname="com.usu.sudokuSolver.board.test.SudokuSolver16x16Test" time="0.012"/>
    <testcase name="testContainDuplicateColumn_NullValue" classname="com.usu.sudokuSolver.board.test.SudokuSolver16x16Test" time="0.025"/>
    <testcase name="testIsSolved_NullValues_WrongValue" classname="com.usu.sudokuSolver.board.test.SudokuSolver16x16Test" time="0.013"/>
    <testcase name="testContainOutlier_CorrectValue" classname="com.usu.sudokuSolver.board.test.SudokuSolver16x16Test" time="0.025"/>
    <testcase name="testContainOutlier_NullValue" classname="com.usu.sudokuSolver.board.test.SudokuSolver16x16Test" time="0.0"/>
    <testcase name="testContainDuplicateRow_CorrectValue" classname="com.usu.sudokuSolver.board.test.SudokuSolver16x16Test" time="0.078"/>
    <testcase name="testContainDuplicateRow_NullValue" classname="com.usu.sudokuSolver.board.test.SudokuSolver16x16Test" time="0.029"/>
  </testsuite>
  <testsuite name="com.usu.sudokuSolver.board.test.SudokuSolver25x25Test" time="0.27">
    <testcase name="testSizeColumn_CorrectValue" classname="com.usu.sudokuSolver.board.test.SudokuSolver25x25Test" time="0.01">
      <error>java.lang.ArrayIndexOutOfBoundsException: 25&#13;
	at com.usu.sudokuSolver.board.SudokuBoardParser.parse(SudokuBoardParser.java:39)&#13;
	at com.usu.sudokuSolver.board.test.SudokuSolver25x25Test.&lt;init&gt;(SudokuSolver25x25Test.java:102)&#13;
	at sun.reflect.NativeConstructorAccessorImpl.newInstance0(Native Method)&#13;
	at sun.reflect.NativeConstructorAccessorImpl.newInstance(Unknown Source)&#13;
	at sun.reflect.DelegatingConstructorAccessorImpl.newInstance(Unknown Source)&#13;
	at java.lang.reflect.Constructor.newInstance(Unknown Source)&#13;
	at org.junit.runners.BlockJUnit4ClassRunner.createTest(BlockJUnit4ClassRunner.java:217)&#13;
	at org.junit.runners.BlockJUnit4ClassRunner$1.runReflectiveCall(BlockJUnit4ClassRunner.java:266)&#13;
	at org.junit.internal.runners.model.ReflectiveCallable.run(ReflectiveCallable.java:12)&#13;
	at org.junit.runners.BlockJUnit4ClassRunner.methodBlock(BlockJUnit4ClassRunner.java:263)&#13;
	at org.junit.runners.BlockJUnit4ClassRunner.runChild(BlockJUnit4ClassRunner.java:78)&#13;
	at org.junit.runners.BlockJUnit4ClassRunner.runChild(BlockJUnit4ClassRunner.java:57)&#13;
	at org.junit.runners.ParentRunner$3.run(ParentRunner.java:290)&#13;
	at org.junit.runners.ParentRunner$1.schedule(ParentRunner.java:71)&#13;
	at org.junit.runners.ParentRunner.runChildren(ParentRunner.java:288)&#13;
	at org.junit.runners.ParentRunner.access$000(ParentRunner.java:58)&#13;
	at org.junit.runners.ParentRunner$2.evaluate(ParentRunner.java:268)&#13;
	at org.junit.runners.ParentRunner.run(ParentRunner.java:363)&#13;
	at org.eclipse.jdt.internal.junit4.runner.JUnit4TestReference.run(JUnit4TestReference.java:86)&#13;
	at org.eclipse.jdt.internal.junit.runner.TestExecution.run(TestExecution.java:38)&#13;
	at org.eclipse.jdt.internal.junit.runner.RemoteTestRunner.runTests(RemoteTestRunner.java:459)&#13;
	at org.eclipse.jdt.internal.junit.runner.RemoteTestRunner.runTests(RemoteTestRunner.java:678)&#13;
	at org.eclipse.jdt.internal.junit.runner.RemoteTestRunner.run(RemoteTestRunner.java:382)&#13;
	at org.eclipse.jdt.internal.junit.runner.RemoteTestRunner.main(RemoteTestRunner.java:192)&#13;
&#13;
</error>
    </testcase>
    <testcase name="testIsSolved_NullValues" classname="com.usu.sudokuSolver.board.test.SudokuSolver25x25Test" time="0.008">
      <error>java.lang.ArrayIndexOutOfBoundsException: 25&#13;
	at com.usu.sudokuSolver.board.SudokuBoardParser.parse(SudokuBoardParser.java:39)&#13;
	at com.usu.sudokuSolver.board.test.SudokuSolver25x25Test.&lt;init&gt;(SudokuSolver25x25Test.java:102)&#13;
	at sun.reflect.NativeConstructorAccessorImpl.newInstance0(Native Method)&#13;
	at sun.reflect.NativeConstructorAccessorImpl.newInstance(Unknown Source)&#13;
	at sun.reflect.DelegatingConstructorAccessorImpl.newInstance(Unknown Source)&#13;
	at java.lang.reflect.Constructor.newInstance(Unknown Source)&#13;
	at org.junit.runners.BlockJUnit4ClassRunner.createTest(BlockJUnit4ClassRunner.java:217)&#13;
	at org.junit.runners.BlockJUnit4ClassRunner$1.runReflectiveCall(BlockJUnit4ClassRunner.java:266)&#13;
	at org.junit.internal.runners.model.ReflectiveCallable.run(ReflectiveCallable.java:12)&#13;
	at org.junit.runners.BlockJUnit4ClassRunner.methodBlock(BlockJUnit4ClassRunner.java:263)&#13;
	at org.junit.runners.BlockJUnit4ClassRunner.runChild(BlockJUnit4ClassRunner.java:78)&#13;
	at org.junit.runners.BlockJUnit4ClassRunner.runChild(BlockJUnit4ClassRunner.java:57)&#13;
	at org.junit.runners.ParentRunner$3.run(ParentRunner.java:290)&#13;
	at org.junit.runners.ParentRunner$1.schedule(ParentRunner.java:71)&#13;
	at org.junit.runners.ParentRunner.runChildren(ParentRunner.java:288)&#13;
	at org.junit.runners.ParentRunner.access$000(ParentRunner.java:58)&#13;
	at org.junit.runners.ParentRunner$2.evaluate(ParentRunner.java:268)&#13;
	at org.junit.runners.ParentRunner.run(ParentRunner.java:363)&#13;
	at org.eclipse.jdt.internal.junit4.runner.JUnit4TestReference.run(JUnit4TestReference.java:86)&#13;
	at org.eclipse.jdt.internal.junit.runner.TestExecution.run(TestExecution.java:38)&#13;
	at org.eclipse.jdt.internal.junit.runner.RemoteTestRunner.runTests(RemoteTestRunner.java:459)&#13;
	at org.eclipse.jdt.internal.junit.runner.RemoteTestRunner.runTests(RemoteTestRunner.java:678)&#13;
	at org.eclipse.jdt.internal.junit.runner.RemoteTestRunner.run(RemoteTestRunner.java:382)&#13;
	at org.eclipse.jdt.internal.junit.runner.RemoteTestRunner.main(RemoteTestRunner.java:192)&#13;
&#13;
</error>
    </testcase>
    <testcase name="testContainDuplicateColumn_CorrectValue" classname="com.usu.sudokuSolver.board.test.SudokuSolver25x25Test" time="0.001">
      <error>java.lang.ArrayIndexOutOfBoundsException: 25&#13;
	at com.usu.sudokuSolver.board.SudokuBoardParser.parse(SudokuBoardParser.java:39)&#13;
	at com.usu.sudokuSolver.board.test.SudokuSolver25x25Test.&lt;init&gt;(SudokuSolver25x25Test.java:102)&#13;
	at sun.reflect.NativeConstructorAccessorImpl.newInstance0(Native Method)&#13;
	at sun.reflect.NativeConstructorAccessorImpl.newInstance(Unknown Source)&#13;
	at sun.reflect.DelegatingConstructorAccessorImpl.newInstance(Unknown Source)&#13;
	at java.lang.reflect.Constructor.newInstance(Unknown Source)&#13;
	at org.junit.runners.BlockJUnit4ClassRunner.createTest(BlockJUnit4ClassRunner.java:217)&#13;
	at org.junit.runners.BlockJUnit4ClassRunner$1.runReflectiveCall(BlockJUnit4ClassRunner.java:266)&#13;
	at org.junit.internal.runners.model.ReflectiveCallable.run(ReflectiveCallable.java:12)&#13;
	at org.junit.runners.BlockJUnit4ClassRunner.methodBlock(BlockJUnit4ClassRunner.java:263)&#13;
	at org.junit.runners.BlockJUnit4ClassRunner.runChild(BlockJUnit4ClassRunner.java:78)&#13;
	at org.junit.runners.BlockJUnit4ClassRunner.runChild(BlockJUnit4ClassRunner.java:57)&#13;
	at org.junit.runners.ParentRunner$3.run(ParentRunner.java:290)&#13;
	at org.junit.runners.ParentRunner$1.schedule(ParentRunner.java:71)&#13;
	at org.junit.runners.ParentRunner.runChildren(ParentRunner.java:288)&#13;
	at org.junit.runners.ParentRunner.access$000(ParentRunner.java:58)&#13;
	at org.junit.runners.ParentRunner$2.evaluate(ParentRunner.java:268)&#13;
	at org.junit.runners.ParentRunner.run(ParentRunner.java:363)&#13;
	at org.eclipse.jdt.internal.junit4.runner.JUnit4TestReference.run(JUnit4TestReference.java:86)&#13;
	at org.eclipse.jdt.internal.junit.runner.TestExecution.run(TestExecution.java:38)&#13;
	at org.eclipse.jdt.internal.junit.runner.RemoteTestRunner.runTests(RemoteTestRunner.java:459)&#13;
	at org.eclipse.jdt.internal.junit.runner.RemoteTestRunner.runTests(RemoteTestRunner.java:678)&#13;
	at org.eclipse.jdt.internal.junit.runner.RemoteTestRunner.run(RemoteTestRunner.java:382)&#13;
	at org.eclipse.jdt.internal.junit.runner.RemoteTestRunner.main(RemoteTestRunner.java:192)&#13;
&#13;
</error>
    </testcase>
    <testcase name="testSizeRow_CorrectValue" classname="com.usu.sudokuSolver.board.test.SudokuSolver25x25Test" time="0.0">
      <error>java.lang.ArrayIndexOutOfBoundsException: 25&#13;
	at com.usu.sudokuSolver.board.SudokuBoardParser.parse(SudokuBoardParser.java:39)&#13;
	at com.usu.sudokuSolver.board.test.SudokuSolver25x25Test.&lt;init&gt;(SudokuSolver25x25Test.java:102)&#13;
	at sun.reflect.NativeConstructorAccessorImpl.newInstance0(Native Method)&#13;
	at sun.reflect.NativeConstructorAccessorImpl.newInstance(Unknown Source)&#13;
	at sun.reflect.DelegatingConstructorAccessorImpl.newInstance(Unknown Source)&#13;
	at java.lang.reflect.Constructor.newInstance(Unknown Source)&#13;
	at org.junit.runners.BlockJUnit4ClassRunner.createTest(BlockJUnit4ClassRunner.java:217)&#13;
	at org.junit.runners.BlockJUnit4ClassRunner$1.runReflectiveCall(BlockJUnit4ClassRunner.java:266)&#13;
	at org.junit.internal.runners.model.ReflectiveCallable.run(ReflectiveCallable.java:12)&#13;
	at org.junit.runners.BlockJUnit4ClassRunner.methodBlock(BlockJUnit4ClassRunner.java:263)&#13;
	at org.junit.runners.BlockJUnit4ClassRunner.runChild(BlockJUnit4ClassRunner.java:78)&#13;
	at org.junit.runners.BlockJUnit4ClassRunner.runChild(BlockJUnit4ClassRunner.java:57)&#13;
	at org.junit.runners.ParentRunner$3.run(ParentRunner.java:290)&#13;
	at org.junit.runners.ParentRunner$1.schedule(ParentRunner.java:71)&#13;
	at org.junit.runners.ParentRunner.runChildren(ParentRunner.java:288)&#13;
	at org.junit.runners.ParentRunner.access$000(ParentRunner.java:58)&#13;
	at org.junit.runners.ParentRunner$2.evaluate(ParentRunner.java:268)&#13;
	at org.junit.runners.ParentRunner.run(ParentRunner.java:363)&#13;
	at org.eclipse.jdt.internal.junit4.runner.JUnit4TestReference.run(JUnit4TestReference.java:86)&#13;
	at org.eclipse.jdt.internal.junit.runner.TestExecution.run(TestExecution.java:38)&#13;
	at org.eclipse.jdt.internal.junit.runner.RemoteTestRunner.runTests(RemoteTestRunner.java:459)&#13;
	at org.eclipse.jdt.internal.junit.runner.RemoteTestRunner.runTests(RemoteTestRunner.java:678)&#13;
	at org.eclipse.jdt.internal.junit.runner.RemoteTestRunner.run(RemoteTestRunner.java:382)&#13;
	at org.eclipse.jdt.internal.junit.runner.RemoteTestRunner.main(RemoteTestRunner.java:192)&#13;
&#13;
</error>
    </testcase>
    <testcase name="testContainDuplicateColumn_NullValue" classname="com.usu.sudokuSolver.board.test.SudokuSolver25x25Test" time="0.0">
      <error>java.lang.ArrayIndexOutOfBoundsException: 25&#13;
	at com.usu.sudokuSolver.board.SudokuBoardParser.parse(SudokuBoardParser.java:39)&#13;
	at com.usu.sudokuSolver.board.test.SudokuSolver25x25Test.&lt;init&gt;(SudokuSolver25x25Test.java:102)&#13;
	at sun.reflect.NativeConstructorAccessorImpl.newInstance0(Native Method)&#13;
	at sun.reflect.NativeConstructorAccessorImpl.newInstance(Unknown Source)&#13;
	at sun.reflect.DelegatingConstructorAccessorImpl.newInstance(Unknown Source)&#13;
	at java.lang.reflect.Constructor.newInstance(Unknown Source)&#13;
	at org.junit.runners.BlockJUnit4ClassRunner.createTest(BlockJUnit4ClassRunner.java:217)&#13;
	at org.junit.runners.BlockJUnit4ClassRunner$1.runReflectiveCall(BlockJUnit4ClassRunner.java:266)&#13;
	at org.junit.internal.runners.model.ReflectiveCallable.run(ReflectiveCallable.java:12)&#13;
	at org.junit.runners.BlockJUnit4ClassRunner.methodBlock(BlockJUnit4ClassRunner.java:263)&#13;
	at org.junit.runners.BlockJUnit4ClassRunner.runChild(BlockJUnit4ClassRunner.java:78)&#13;
	at org.junit.runners.BlockJUnit4ClassRunner.runChild(BlockJUnit4ClassRunner.java:57)&#13;
	at org.junit.runners.ParentRunner$3.run(ParentRunner.java:290)&#13;
	at org.junit.runners.ParentRunner$1.schedule(ParentRunner.java:71)&#13;
	at org.junit.runners.ParentRunner.runChildren(ParentRunner.java:288)&#13;
	at org.junit.runners.ParentRunner.access$000(ParentRunner.java:58)&#13;
	at org.junit.runners.ParentRunner$2.evaluate(ParentRunner.java:268)&#13;
	at org.junit.runners.ParentRunner.run(ParentRunner.java:363)&#13;
	at org.eclipse.jdt.internal.junit4.runner.JUnit4TestReference.run(JUnit4TestReference.java:86)&#13;
	at org.eclipse.jdt.internal.junit.runner.TestExecution.run(TestExecution.java:38)&#13;
	at org.eclipse.jdt.internal.junit.runner.RemoteTestRunner.runTests(RemoteTestRunner.java:459)&#13;
	at org.eclipse.jdt.internal.junit.runner.RemoteTestRunner.runTests(RemoteTestRunner.java:678)&#13;
	at org.eclipse.jdt.internal.junit.runner.RemoteTestRunner.run(RemoteTestRunner.java:382)&#13;
	at org.eclipse.jdt.internal.junit.runner.RemoteTestRunner.main(RemoteTestRunner.java:192)&#13;
&#13;
</error>
    </testcase>
    <testcase name="testIsSolved_NullValues_WrongValue" classname="com.usu.sudokuSolver.board.test.SudokuSolver25x25Test" time="0.001">
      <error>java.lang.ArrayIndexOutOfBoundsException: 25&#13;
	at com.usu.sudokuSolver.board.SudokuBoardParser.parse(SudokuBoardParser.java:39)&#13;
	at com.usu.sudokuSolver.board.test.SudokuSolver25x25Test.&lt;init&gt;(SudokuSolver25x25Test.java:102)&#13;
	at sun.reflect.NativeConstructorAccessorImpl.newInstance0(Native Method)&#13;
	at sun.reflect.NativeConstructorAccessorImpl.newInstance(Unknown Source)&#13;
	at sun.reflect.DelegatingConstructorAccessorImpl.newInstance(Unknown Source)&#13;
	at java.lang.reflect.Constructor.newInstance(Unknown Source)&#13;
	at org.junit.runners.BlockJUnit4ClassRunner.createTest(BlockJUnit4ClassRunner.java:217)&#13;
	at org.junit.runners.BlockJUnit4ClassRunner$1.runReflectiveCall(BlockJUnit4ClassRunner.java:266)&#13;
	at org.junit.internal.runners.model.ReflectiveCallable.run(ReflectiveCallable.java:12)&#13;
	at org.junit.runners.BlockJUnit4ClassRunner.methodBlock(BlockJUnit4ClassRunner.java:263)&#13;
	at org.junit.runners.BlockJUnit4ClassRunner.runChild(BlockJUnit4ClassRunner.java:78)&#13;
	at org.junit.runners.BlockJUnit4ClassRunner.runChild(BlockJUnit4ClassRunner.java:57)&#13;
	at org.junit.runners.ParentRunner$3.run(ParentRunner.java:290)&#13;
	at org.junit.runners.ParentRunner$1.schedule(ParentRunner.java:71)&#13;
	at org.junit.runners.ParentRunner.runChildren(ParentRunner.java:288)&#13;
	at org.junit.runners.ParentRunner.access$000(ParentRunner.java:58)&#13;
	at org.junit.runners.ParentRunner$2.evaluate(ParentRunner.java:268)&#13;
	at org.junit.runners.ParentRunner.run(ParentRunner.java:363)&#13;
	at org.eclipse.jdt.internal.junit4.runner.JUnit4TestReference.run(JUnit4TestReference.java:86)&#13;
	at org.eclipse.jdt.internal.junit.runner.TestExecution.run(TestExecution.java:38)&#13;
	at org.eclipse.jdt.internal.junit.runner.RemoteTestRunner.runTests(RemoteTestRunner.java:459)&#13;
	at org.eclipse.jdt.internal.junit.runner.RemoteTestRunner.runTests(RemoteTestRunner.java:678)&#13;
	at org.eclipse.jdt.internal.junit.runner.RemoteTestRunner.run(RemoteTestRunner.java:382)&#13;
	at org.eclipse.jdt.internal.junit.runner.RemoteTestRunner.main(RemoteTestRunner.java:192)&#13;
&#13;
</error>
    </testcase>
    <testcase name="testContainOutlier_CorrectValue" classname="com.usu.sudokuSolver.board.test.SudokuSolver25x25Test" time="0.0">
      <error>java.lang.ArrayIndexOutOfBoundsException: 25&#13;
	at com.usu.sudokuSolver.board.SudokuBoardParser.parse(SudokuBoardParser.java:39)&#13;
	at com.usu.sudokuSolver.board.test.SudokuSolver25x25Test.&lt;init&gt;(SudokuSolver25x25Test.java:102)&#13;
	at sun.reflect.NativeConstructorAccessorImpl.newInstance0(Native Method)&#13;
	at sun.reflect.NativeConstructorAccessorImpl.newInstance(Unknown Source)&#13;
	at sun.reflect.DelegatingConstructorAccessorImpl.newInstance(Unknown Source)&#13;
	at java.lang.reflect.Constructor.newInstance(Unknown Source)&#13;
	at org.junit.runners.BlockJUnit4ClassRunner.createTest(BlockJUnit4ClassRunner.java:217)&#13;
	at org.junit.runners.BlockJUnit4ClassRunner$1.runReflectiveCall(BlockJUnit4ClassRunner.java:266)&#13;
	at org.junit.internal.runners.model.ReflectiveCallable.run(ReflectiveCallable.java:12)&#13;
	at org.junit.runners.BlockJUnit4ClassRunner.methodBlock(BlockJUnit4ClassRunner.java:263)&#13;
	at org.junit.runners.BlockJUnit4ClassRunner.runChild(BlockJUnit4ClassRunner.java:78)&#13;
	at org.junit.runners.BlockJUnit4ClassRunner.runChild(BlockJUnit4ClassRunner.java:57)&#13;
	at org.junit.runners.ParentRunner$3.run(ParentRunner.java:290)&#13;
	at org.junit.runners.ParentRunner$1.schedule(ParentRunner.java:71)&#13;
	at org.junit.runners.ParentRunner.runChildren(ParentRunner.java:288)&#13;
	at org.junit.runners.ParentRunner.access$000(ParentRunner.java:58)&#13;
	at org.junit.runners.ParentRunner$2.evaluate(ParentRunner.java:268)&#13;
	at org.junit.runners.ParentRunner.run(ParentRunner.java:363)&#13;
	at org.eclipse.jdt.internal.junit4.runner.JUnit4TestReference.run(JUnit4TestReference.java:86)&#13;
	at org.eclipse.jdt.internal.junit.runner.TestExecution.run(TestExecution.java:38)&#13;
	at org.eclipse.jdt.internal.junit.runner.RemoteTestRunner.runTests(RemoteTestRunner.java:459)&#13;
	at org.eclipse.jdt.internal.junit.runner.RemoteTestRunner.runTests(RemoteTestRunner.java:678)&#13;
	at org.eclipse.jdt.internal.junit.runner.RemoteTestRunner.run(RemoteTestRunner.java:382)&#13;
	at org.eclipse.jdt.internal.junit.runner.RemoteTestRunner.main(RemoteTestRunner.java:192)&#13;
&#13;
</error>
    </testcase>
    <testcase name="testContainOutlier_NullValue" classname="com.usu.sudokuSolver.board.test.SudokuSolver25x25Test" time="0.008">
      <error>java.lang.ArrayIndexOutOfBoundsException: 25&#13;
	at com.usu.sudokuSolver.board.SudokuBoardParser.parse(SudokuBoardParser.java:39)&#13;
	at com.usu.sudokuSolver.board.test.SudokuSolver25x25Test.&lt;init&gt;(SudokuSolver25x25Test.java:102)&#13;
	at sun.reflect.NativeConstructorAccessorImpl.newInstance0(Native Method)&#13;
	at sun.reflect.NativeConstructorAccessorImpl.newInstance(Unknown Source)&#13;
	at sun.reflect.DelegatingConstructorAccessorImpl.newInstance(Unknown Source)&#13;
	at java.lang.reflect.Constructor.newInstance(Unknown Source)&#13;
	at org.junit.runners.BlockJUnit4ClassRunner.createTest(BlockJUnit4ClassRunner.java:217)&#13;
	at org.junit.runners.BlockJUnit4ClassRunner$1.runReflectiveCall(BlockJUnit4ClassRunner.java:266)&#13;
	at org.junit.internal.runners.model.ReflectiveCallable.run(ReflectiveCallable.java:12)&#13;
	at org.junit.runners.BlockJUnit4ClassRunner.methodBlock(BlockJUnit4ClassRunner.java:263)&#13;
	at org.junit.runners.BlockJUnit4ClassRunner.runChild(BlockJUnit4ClassRunner.java:78)&#13;
	at org.junit.runners.BlockJUnit4ClassRunner.runChild(BlockJUnit4ClassRunner.java:57)&#13;
	at org.junit.runners.ParentRunner$3.run(ParentRunner.java:290)&#13;
	at org.junit.runners.ParentRunner$1.schedule(ParentRunner.java:71)&#13;
	at org.junit.runners.ParentRunner.runChildren(ParentRunner.java:288)&#13;
	at org.junit.runners.ParentRunner.access$000(ParentRunner.java:58)&#13;
	at org.junit.runners.ParentRunner$2.evaluate(ParentRunner.java:268)&#13;
	at org.junit.runners.ParentRunner.run(ParentRunner.java:363)&#13;
	at org.eclipse.jdt.internal.junit4.runner.JUnit4TestReference.run(JUnit4TestReference.java:86)&#13;
	at org.eclipse.jdt.internal.junit.runner.TestExecution.run(TestExecution.java:38)&#13;
	at org.eclipse.jdt.internal.junit.runner.RemoteTestRunner.runTests(RemoteTestRunner.java:459)&#13;
	at org.eclipse.jdt.internal.junit.runner.RemoteTestRunner.runTests(RemoteTestRunner.java:678)&#13;
	at org.eclipse.jdt.internal.junit.runner.RemoteTestRunner.run(RemoteTestRunner.java:382)&#13;
	at org.eclipse.jdt.internal.junit.runner.RemoteTestRunner.main(RemoteTestRunner.java:192)&#13;
&#13;
</error>
    </testcase>
    <testcase name="testContainDuplicateRow_CorrectValue" classname="com.usu.sudokuSolver.board.test.SudokuSolver25x25Test" time="0.0">
      <error>java.lang.ArrayIndexOutOfBoundsException: 25&#13;
	at com.usu.sudokuSolver.board.SudokuBoardParser.parse(SudokuBoardParser.java:39)&#13;
	at com.usu.sudokuSolver.board.test.SudokuSolver25x25Test.&lt;init&gt;(SudokuSolver25x25Test.java:102)&#13;
	at sun.reflect.NativeConstructorAccessorImpl.newInstance0(Native Method)&#13;
	at sun.reflect.NativeConstructorAccessorImpl.newInstance(Unknown Source)&#13;
	at sun.reflect.DelegatingConstructorAccessorImpl.newInstance(Unknown Source)&#13;
	at java.lang.reflect.Constructor.newInstance(Unknown Source)&#13;
	at org.junit.runners.BlockJUnit4ClassRunner.createTest(BlockJUnit4ClassRunner.java:217)&#13;
	at org.junit.runners.BlockJUnit4ClassRunner$1.runReflectiveCall(BlockJUnit4ClassRunner.java:266)&#13;
	at org.junit.internal.runners.model.ReflectiveCallable.run(ReflectiveCallable.java:12)&#13;
	at org.junit.runners.BlockJUnit4ClassRunner.methodBlock(BlockJUnit4ClassRunner.java:263)&#13;
	at org.junit.runners.BlockJUnit4ClassRunner.runChild(BlockJUnit4ClassRunner.java:78)&#13;
	at org.junit.runners.BlockJUnit4ClassRunner.runChild(BlockJUnit4ClassRunner.java:57)&#13;
	at org.junit.runners.ParentRunner$3.run(ParentRunner.java:290)&#13;
	at org.junit.runners.ParentRunner$1.schedule(ParentRunner.java:71)&#13;
	at org.junit.runners.ParentRunner.runChildren(ParentRunner.java:288)&#13;
	at org.junit.runners.ParentRunner.access$000(ParentRunner.java:58)&#13;
	at org.junit.runners.ParentRunner$2.evaluate(ParentRunner.java:268)&#13;
	at org.junit.runners.ParentRunner.run(ParentRunner.java:363)&#13;
	at org.eclipse.jdt.internal.junit4.runner.JUnit4TestReference.run(JUnit4TestReference.java:86)&#13;
	at org.eclipse.jdt.internal.junit.runner.TestExecution.run(TestExecution.java:38)&#13;
	at org.eclipse.jdt.internal.junit.runner.RemoteTestRunner.runTests(RemoteTestRunner.java:459)&#13;
	at org.eclipse.jdt.internal.junit.runner.RemoteTestRunner.runTests(RemoteTestRunner.java:678)&#13;
	at org.eclipse.jdt.internal.junit.runner.RemoteTestRunner.run(RemoteTestRunner.java:382)&#13;
	at org.eclipse.jdt.internal.junit.runner.RemoteTestRunner.main(RemoteTestRunner.java:192)&#13;
&#13;
</error>
    </testcase>
    <testcase name="testContainDuplicateRow_NullValue" classname="com.usu.sudokuSolver.board.test.SudokuSolver25x25Test" time="0.001">
      <error>java.lang.ArrayIndexOutOfBoundsException: 25&#13;
	at com.usu.sudokuSolver.board.SudokuBoardParser.parse(SudokuBoardParser.java:39)&#13;
	at com.usu.sudokuSolver.board.test.SudokuSolver25x25Test.&lt;init&gt;(SudokuSolver25x25Test.java:102)&#13;
	at sun.reflect.NativeConstructorAccessorImpl.newInstance0(Native Method)&#13;
	at sun.reflect.NativeConstructorAccessorImpl.newInstance(Unknown Source)&#13;
	at sun.reflect.DelegatingConstructorAccessorImpl.newInstance(Unknown Source)&#13;
	at java.lang.reflect.Constructor.newInstance(Unknown Source)&#13;
	at org.junit.runners.BlockJUnit4ClassRunner.createTest(BlockJUnit4ClassRunner.java:217)&#13;
	at org.junit.runners.BlockJUnit4ClassRunner$1.runReflectiveCall(BlockJUnit4ClassRunner.java:266)&#13;
	at org.junit.internal.runners.model.ReflectiveCallable.run(ReflectiveCallable.java:12)&#13;
	at org.junit.runners.BlockJUnit4ClassRunner.methodBlock(BlockJUnit4ClassRunner.java:263)&#13;
	at org.junit.runners.BlockJUnit4ClassRunner.runChild(BlockJUnit4ClassRunner.java:78)&#13;
	at org.junit.runners.BlockJUnit4ClassRunner.runChild(BlockJUnit4ClassRunner.java:57)&#13;
	at org.junit.runners.ParentRunner$3.run(ParentRunner.java:290)&#13;
	at org.junit.runners.ParentRunner$1.schedule(ParentRunner.java:71)&#13;
	at org.junit.runners.ParentRunner.runChildren(ParentRunner.java:288)&#13;
	at org.junit.runners.ParentRunner.access$000(ParentRunner.java:58)&#13;
	at org.junit.runners.ParentRunner$2.evaluate(ParentRunner.java:268)&#13;
	at org.junit.runners.ParentRunner.run(ParentRunner.java:363)&#13;
	at org.eclipse.jdt.internal.junit4.runner.JUnit4TestReference.run(JUnit4TestReference.java:86)&#13;
	at org.eclipse.jdt.internal.junit.runner.TestExecution.run(TestExecution.java:38)&#13;
	at org.eclipse.jdt.internal.junit.runner.RemoteTestRunner.runTests(RemoteTestRunner.java:459)&#13;
	at org.eclipse.jdt.internal.junit.runner.RemoteTestRunner.runTests(RemoteTestRunner.java:678)&#13;
	at org.eclipse.jdt.internal.junit.runner.RemoteTestRunner.run(RemoteTestRunner.java:382)&#13;
	at org.eclipse.jdt.internal.junit.runner.RemoteTestRunner.main(RemoteTestRunner.java:192)&#13;
&#13;
</error>
    </testcase>
  </testsuite>
  <testsuite name="com.usu.sudokuSolver.board.test.SudokuSolver9x9Test" time="1.267">
    <testcase name="testSizeColumn_CorrectValue" classname="com.usu.sudokuSolver.board.test.SudokuSolver9x9Test" time="0.0"/>
    <testcase name="testIsSolved_NullValues" classname="com.usu.sudokuSolver.board.test.SudokuSolver9x9Test" time="0.0"/>
    <testcase name="testContainDuplicateColumn_CorrectValue" classname="com.usu.sudokuSolver.board.test.SudokuSolver9x9Test" time="0.0"/>
    <testcase name="testIsSolved_Correct" classname="com.usu.sudokuSolver.board.test.SudokuSolver9x9Test" time="0.702"/>
    <testcase name="testSizeRow_CorrectValue" classname="com.usu.sudokuSolver.board.test.SudokuSolver9x9Test" time="0.002"/>
    <testcase name="testContainDuplicateColumn_NullValue" classname="com.usu.sudokuSolver.board.test.SudokuSolver9x9Test" time="0.002"/>
    <testcase name="testIsSolved_NullValues_WrongValue" classname="com.usu.sudokuSolver.board.test.SudokuSolver9x9Test" time="0.006"/>
    <testcase name="testContainOutlier_CorrectValue" classname="com.usu.sudokuSolver.board.test.SudokuSolver9x9Test" time="0.002"/>
    <testcase name="testContainOutlier_NullValue" classname="com.usu.sudokuSolver.board.test.SudokuSolver9x9Test" time="0.002"/>
    <testcase name="testIsSolved_Correct_WrongValue" classname="com.usu.sudokuSolver.board.test.SudokuSolver9x9Test" time="0.549"/>
    <testcase name="testContainDuplicateRow_CorrectValue" classname="com.usu.sudokuSolver.board.test.SudokuSolver9x9Test" time="0.001"/>
    <testcase name="testContainDuplicateRow_NullValue" classname="com.usu.sudokuSolver.board.test.SudokuSolver9x9Test" time="0.0"/>
  </testsuite>
</testrun>
